app/models/requirement.rb

```diff
@@ -141,6 +141,8 @@ def to_form_json(requirement_block_key = requirement_block&.key)
 
     json.merge!({ description: hint }) if hint
 
+    json.merge!({ elective: elective }) if elective
```

Comment:
@sunto shouldn't we also add `show: false` by default if a requirement has `elective: true`?  Alternatively it can be done in `[formio-component-traversal.ts]`